+nosnoc/+discrete_time_problem/Stewart.m

```diff
@@ -885,7 +885,7 @@ function create_solver(obj, solver_options, plugin)
                 obj.w.sort_by_index();
                 obj.g.sort_by_index();
 
-                solver_options.assume_lower_bounds = true;
+                
```

Comment:
we had to have this option in the previous PR  to avoid copying constraints. now it is removed and treated somewhere else? (I had added comments in #145 for this option). please clarify

Reply:
resolved, see below, and then declare as resolved